src/main/resources/view/MatchCard.fxml

```diff
@@ -27,7 +27,7 @@
           </minWidth>
         </Label>
         <Label fx:id="name" styleClass="cell_big_label" text="\$first" />
-        <Label fx:id="success" styleClass="cell_big_label" text="\$success" textFill="RED">
+        <Label fx:id="success" styleClass="error" text="\$success" >
```

Comment:
Why `success` case but `styleClass="error"`

Reply:
Because the styleClass="error" styling makes the text red.